pkg/acsengine/k8s_versions.go

```diff
@@ -269,7 +269,7 @@ var KubeConfigs = map[string]map[string]string{
 		"dnsmasq":         "k8s-dns-dnsmasq-nanny-amd64:1.14.4",
 		"pause":           "pause-amd64:3.0",
 		"tiller":          DefaultTillerImage,
-		"windowszip":      "v1.7.0intwinnat.zip",
+		"windowszip":      "v1.7.10-1int.zip ",
```

Comment:
@CecileRobertMichon wrong place. Should be under KubernetesVersion1Dot7Dot10

Reply:
Oops I'm at Azure boot camp so I was multi-tasking... Thank you for catching that. Just pushed to fix.